omigami/spectra_matching/spectra_matching.py

```diff
@@ -214,8 +233,17 @@ def _send_request(payload: Payload, endpoint: str) -> requests.Response:
 
         if api_request.status_code == 401:
             raise InvalidCredentials("Your credentials are invalid.")
-        if api_request.status_code == 404:
+        elif api_request.status_code == 404:
             raise NotFoundError("The API endpoint couldn't be reached.")
+        elif api_request.status_code == 500:
+            log.error(
+                f"ERROR: An error happened during spectra matching. "
+                f"Please try again or contact DataRevenue for more information. "
+                f"The list of spectra that failed can be accessed on the failed_spectra attribute."
+                f"\n\n{api_request.json()['status']['info']}.\n"
+            )
+            self._failed_spectra.extend(batch)
+            return
```

Comment:
instead of raising, we log the error now.

Reply:
ahh this is why OMI-448 data did not raise any errors, and instead just logged it. Should we update it @carloszanella @bernardolk

Reply:
No problems here I think :)

Reply:
ok solved on slack